test/local/metrics/context.js

```diff
@@ -940,44 +940,45 @@ describe('metricsContext', () => {
             flowBeginTime: expectedTime
           }
         }
-      }
-      sinon.stub(Date, 'now').callsFake(function() {
-        return expectedTime + 20000
-      })
+      };
+      sinon.stub(Date, 'now').callsFake(() => {
+        return expectedTime + 20000;
+      });
 
       try {
-        var metricsContext = require(modulePath)(mockLog, mockConfig)
-        var valid = metricsContext.validate.call(mockRequest)
+        const metricsContext = require(modulePath)(mockLog, mockConfig);
+        // eslint-disable-next-line no-var
+        var valid = metricsContext.validate.call(mockRequest);
```

Comment:
@hritvi, for these cases, instead of disabling the rule, can you declare `let valid;` before the `try {` line, then make this just an assignment to that, `valid = ...` ?